docs/install_guides/installing-model-dev-tools.md

```diff
@@ -73,96 +89,36 @@ Activate the newly created Python virtual environment by issuing this command:
 @endsphinxdirective
 
 
-### Step 3. Set Up and Update PIP to the Highest Version
+#### Step 3. Set Up and Update PIP to the Highest Version
 Make sure `pip` is installed in your environment and upgrade it to the latest version by issuing the following command:
 
 ```sh
 python -m pip install --upgrade pip
 ```
 
-### Step 4. Install the Package
+#### Step 4. Install the Package
 
 To install and configure the components of the development package for working with specific frameworks, use the commands below.
```

Comment:
"To install and configure .. together with validated versions of specific frameworks"
I guess this is what really happens?

Reply:
Done